captum/attr/_core/layer/grad_cam.py

```diff
@@ -208,10 +213,16 @@ def attribute(
             for layer_grad in layer_gradients
         )
 
-        scaled_acts = tuple(
-            torch.sum(summed_grad * layer_eval, dim=1, keepdim=True)
-            for summed_grad, layer_eval in zip(summed_grads, layer_evals)
-        )
+        if split_channels:
```

Comment:
It looks like it is useful when `relu_attributions == True` because otherwise we can also sum returned value of the attributions.

Reply:
`split_channels` is independent of `relu_attributions`. They can be applied independently.